src/modules/cmdpal/Microsoft.CmdPal.UI/MainWindow.xaml.cs

```diff
@@ -386,7 +411,9 @@ internal void MainWindow_Activated(object sender, WindowActivatedEventArgs args)
                 return;
             }
 
-            PInvoke.ShowWindow(_hwnd, SHOW_WINDOW_CMD.SW_HIDE);
+            // This will DWM cloak our window:
```

Comment:
Nit if we change the implementation detail inside HideWindow are we going to remember to change this comment?